src/cli/index.ts

```diff
@@ -0,0 +1,11 @@
+import { createCommand } from 'commander';
+import { run } from './run';
+import { setup } from './setup';
+
+export function createStarbaseCli() {
+  return createCommand()
+    .name('yarn starbase')
```

Comment:
Seems like the name should just be `starbase` but nbd.

```suggestion
    .name('starbase')
```